auth/database/Cache.py

```diff
@@ -0,0 +1,74 @@
+# Redis cache configuration
+# powered by flask-redis: https://github.com/underyx/flask-redis
+import conf
+from .flaskAlchemyInit import app, db
+from flask_redis import FlaskRedis
+import redis
+
+if (conf.cacheName == 'redis'):
+    REDIS_URL = ('redis://' + conf.cacheUser + ':' + conf.cachePdw
+                 + '@' + conf.cacheHost + ':6379/' + conf.cacheDatabase)
+    app.config['DBA_URL'] = REDIS_URL
+    redis_store = FlaskRedis(app, config_prefix='DBA', strict=False,
+                             encoding="utf-8", socket_keepalive=True,
+                             charset="utf-8", decode_responses=True)
+
+
+elif (conf.cacheName == 'NOCACHE'):
+    print("Warning. Cache policy set to NOCACHE."
```

Comment:
Its OK to use prints on those warnings? or should we use a logger?

Reply:
Using the logger should be better (depending on the infrastructure, prints will not be "printed" until the master process crashes, making runtime management a tad hard to do.